dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@mcknight/CT-1166#egg=dbt-core&subdirectory=core
```

Comment:
was this supposed to be committed?

Reply:
for now yes, during the test conversion process if you touch adapters as well and are doing them all at once to have ci/cd and test locally well just easy to target the branch with your commit till merge time and post review.

Reply:
Just remove this before merge otherwise looks good!